### DISCUSSION TITLE
Java Edition for Devamped

### DISCUSSION THREAD

**Discussion by Onekai88** [Arxance/Devamped#6] [General]
i've been a fan of this texture pack for along time like since v0.1 alpha 5

But there is a huge problem

i play both on bedrock and java

and i only like the 0.2A10 bec 0.2A9 java test isnt as good and the grass texture looks bad and some other things
So i just really want  devamped to be on java more often bec it my fav texture pack and only one i use

**Comment by Arxance** [comment]
0.2a09 was an java test due to its converted version and old port mixed instead of manually porting it.

This is also a reason why I don't port java edition anyone until devamped is fully finished or when the blocks is done. Unless I'm looking for someone to port DEVAMPED in java edition for me.

The grass has been improved since 0.2a07 but I believe the latest (0.2a10) may be finalized since it looks great enough for me.

**Reply by Onekai88** [reply]
Okay so java edition is gonna be for the realese

Thanks for responding
